packages/DialogActions/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@paprika/dialog-actions",
+  "version": "0.1.0",
+  "description": "DialogActions component",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/DialogActions"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/icon": "^0.2.5",
+    "@paprika/raw-button": "^0.2.2",
+    "@paprika/stylers": "^0.2.2",
+    "@paprika/tokens": "^0.1.9",
+    "class-names": "^1.0.0",
```

Comment:
ohhh we are not using `class-name` anymore, just remove this one from the `package.json`